packages/dashboard/src/components/Wizard/Wizard.styles.ts

```diff
@@ -1,28 +1,31 @@
 import { IWizardStyles, IWizardStyleProps } from './Wizard.types';
 
 export const subwayNavWidthPx = 303;
-
 export const subwayNavPaddingPx = 48;
 
 export const getWizardStyles = (props: IWizardStyleProps): IWizardStyles => {
   return {
     wizardContentNavContainer: {
       display: 'flex',
-      flexDirection: 'row'
+      flexDirection: 'row',
+      height: '100%'
     },
     subwayNavSection: {
       width: `${subwayNavWidthPx}px`,
       paddingLeft: `${subwayNavPaddingPx}px`,
       paddingTop: `${subwayNavPaddingPx}px`,
       paddingBottom: `${subwayNavPaddingPx}px`,
-      borderRight: `1px solid ${props.theme.semanticColors.bodyDivider}`
+      borderRight: `1px solid ${props.theme.semanticColors.bodyDivider}`,
+      overflowY: 'auto'
     },
     contentSection: {
-      paddingTop: `${subwayNavPaddingPx}px`
+      width: '100%',

```

Comment:
I don't think we need this with flex. Speaking of which, Grid might actually make more sense for the Wizard.